webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/ncf/EditorTechnique.scala

```diff
@@ -188,7 +188,7 @@ object ParameterType {
       (paramType,agentType) match {
         case (Raw,_) => Right(value)
         case (StringParameter | HereString, AgentType.CfeCommunity | AgentType.CfeEnterprise) =>
-          Right(s""""${value.replaceAll("""(?<!\\)"""", """\\"""").replaceAll("""(?<!\\)\\(?!(\\|"|'))""","""\\\\""")}"""")
+          Right(s""""${value.replaceAll("""\\""", """\\\\""").replaceAll(""""""" , """\\"""" )}"""")
```

Comment:
maybe for more readability we can so it in replaceall

value.replaceAll("(\\|\")", "\\$1")

or 
value.replaceAll("""(\\|")""", "\\$1")

Reply:
github escapes my \ ... i had to put four 4 to display 2 ....

Reply:
is it more readable ?